### PR TITLE
【Hackathon 7th No.20】为 Paddle 新增 Tensor.set_ -part

### PR DESCRIPTION
**Pull request by NKNaN**
refer to: https://github.com/PaddlePaddle/Paddle/pull/68681

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.set_.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+其中 PyTorch 的 `storage_offset` 与 Paddle 的 `offset` 用法不一致，具体如下：
+
+### 参数映射
+
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| source           | source            | 设置的目标 Tensor。               |
+| storage_offset           | offset            | 表示距离内存数据起始位置的偏移量。Pytorch 是指偏移数字的个数，而 Paddle 是指偏移数字对应存储位置的偏移量（以 byte 为单位），需要转写。             |
+| size   | shape            | 设置的目标形状, 仅参数名不一致。    |
+| stride   | stride            | 设置的目标步长,。    |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.set_.md]: 格式有点问题

Comment (NKNaN) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.set_.md]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM for docs

**Pull request merged**